Sources/Public/Animation/LottieView.swift

```diff
@@ -46,12 +52,18 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
 
   /// Creates a `LottieView` that asynchronously loads and displays the given `DotLottieFile`.
   /// The `loadDotLottieFile` closure is called exactly once in `onAppear`.
-  public init(_ loadDotLottieFile: @escaping () async throws -> DotLottieFile?) where Placeholder == EmptyView {
-    self.init(loadDotLottieFile, placeholder: EmptyView.init)
+  /// If you wish to call `loadAnimation` at a different time, you can use the `loadAnimationTrigger` chain function.
+  public init(
+    _ loadDotLottieFile: @escaping () async throws -> DotLottieFile?) where Placeholder == EmptyView
+  {
+    self.init(
+      loadDotLottieFile,
+      placeholder: EmptyView.init)
   }
 
   /// Creates a `LottieView` that asynchronously loads and displays the given `DotLottieFile`.
   /// The `loadDotLottieFile` closure is called exactly once in `onAppear`.
+  /// If you wish to call `loadAnimation` at a different time, you can use the `loadAnimationTrigger` chain function.
```

Comment:
Suggestion that would apply to all of the inits with this sentence:

```suggestion
  /// If you wish to call `loadAnimation` again at a different time, you can use `.loadAnimationTrigger(...)`.
```